float64 yaw
float64 pitch 
float64 roll
float64 acc_x
float64 acc_y
float64 acc_z
float64 gyr_x
float64 gyr_y
float64 gyr_z
float64 mag_x
float64 mag_y
float64 mag_z
